index.bs

```diff
@@ -282,6 +284,20 @@ interface IntersectionObserver {
 		passed to the {{IntersectionObserver}} constructor.  If no
 		{{IntersectionObserverInit/rootMargin}} was passed to the {{IntersectionObserver}}
 		constructor, the value of this attribute is "0px 0px 0px 0px".
+	: <dfn>scrollMargin</dfn>
+	::
+		Offsets are applied to <a>scrollports</a> on the path from <a>intersection root</a> to <a for="IntersectionObserver">target</a>,
+		effectively growing or shrinking the boxes that are used to calculate intersections.
```

Comment:
When this was first discussed with other implementations, we had decided to apply this to the bounding rect of the clip, rather than the border-box ([discussion](https://github.com/w3c/IntersectionObserver/issues/431#issuecomment-1542502858)). Is that implied here? If not, can you include something about that?